docs/adding-translations.md

```diff
@@ -0,0 +1,17 @@
+# Adding translations
+
+There are translatable strings from both JS an Python.
+
+We use [Transifex][transifex] for translations on the project.
+First, you have to set a [~/.transifexrc][transifexrc].
+
+The command `inv i18n` retrieve all these strings locally that you can then push to Transifex with the command `tx push -s`.
```

Comment:
May be we can add the pasted mail content on gitter because `tx push` should absolutely not be donne on branches